### PR TITLE
Feature/SOF-5318

### PR DESCRIPTION
**Pull request by dmitry-spodarets**

### CODE REVIEW

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -8,17 +8,26 @@ jobs:
...
+      - run: aws s3 sync site/ s3://docs.mt3ra.com/
+        shell: bash
         env:
-          S3_LOCATION: ${{ secrets.S3_LOCATION }}
-          S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
-          S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
-          S3_BUCKET: ${{ secrets.S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.S3_ACCESS_KEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_SECRET_KEY }}
+          AWS_DEFAULT_REGION: ${{ secrets.S3_LOCATION }}
```
[the remark this answers is not in the document]
Comment (timurbazhirov) [reply .github/workflows/update.yml:33]: Let's add a note about these to https://exabyte.atlassian.net/wiki/spaces/~660192581/pages/2419982345/Public+Documentations pls.

Same as https://github.com/Exabyte-io/swaggerui/pull/2/files#r774852330.

File: extra/js/ga.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+gtag('config', 'UA-69270713-5');
```
Comment (timurbazhirov) [inline extra/js/ga.js:4]: Was google analytics not included inside other static assets (e.g. `index.html`) previously?

File: .github/workflows/update.yml [hunk 3]
```diff
@@ -8,17 +8,26 @@ jobs:
...
           python-version: 'pypy-2.7'
 
-      - name: Build pages and publish to S3
-        uses: dmitry-spodarets/mkdocs-s3-publish-action@main
+      - name: Build pages
+        uses: Exabyte-io/mkdocs-s3-publish-action@main
+
+      - name: S3 Sync
+        uses: unfor19/install-aws-cli-action@master
+      - run: aws s3 sync site/ s3://docs.mt3ra.com/
```
Comment (timurbazhirov) [inline .github/workflows/update.yml:28]: Didn't the previous action already include the publish step? https://github.com/Exabyte-io/mkdocs-s3-publish-action/commit/6d96ff626877d167e5b578fa645ce95d61ef31e1

File: extra/js/ga.js (hunk 2 above)
Comment (dmitry-spodarets) [reply extra/js/ga.js:4]: From `mkdocs-s3-publish-action` the sync step has been removed due to minio client, so we have the next step for that. It works better this way.

File: .github/workflows/update.yml (hunk 3 above)
Comment (dmitry-spodarets) [reply .github/workflows/update.yml:28]: From mkdocs-s3-publish-action the sync step has been removed due to minio client, so we have the next step for that. It works better this way.

Comment (dmitry-spodarets) [reply .github/workflows/update.yml:28]: GA added to all pages automatically

File: extra/js/ga.js (hunk 2 above)
Comment (dmitry-spodarets) [reply extra/js/ga.js:4]: GA added to all pages automatically

File: .github/workflows/update.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+      - name: Checkout LFS objects
+        run: git lfs pull
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 'pypy-2.7'
+
+      - name: Build pages
+        uses: Exabyte-io/mkdocs-s3-publish-action@main
```
Comment (timurbazhirov) [inline .github/workflows/update.yml:24]: Per our discussion, let's fork the repos as:
1. Exabyte-io/action-mkdocs-s3-publish@main
2. Exabyte-io/action-install-aws-cli@main

Comment (dmitry-spodarets) [reply .github/workflows/update.yml:24]: done.

Comment (timurbazhirov) [reply .github/workflows/update.yml:24]: `Exabyte-io/mkdocs-s3-publish-action@main` should be `Exabyte-io/action-mkdocs-s3-publish@main`

File: extra/js/ga.js (hunk 2 above)
Comment (timurbazhirov) [reply extra/js/ga.js:4]: Seems like a duplicate with https://github.com/Exabyte-io/documentation/pull/218/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R39 below - do we need to add the ID in both places??

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**